packages/core/tests/integration/components/navi-visualization-config/apex-pie-test.js

```diff
@@ -0,0 +1,18 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import { render } from '@ember/test-helpers';
+
+module('Integration | Component | navi visualization config - apex-pie', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
```

Comment:
```suggestion
    assert.expect(1);
```
We should use `assert.expect` at the beginning of tests where we have some asynchronous behavior